components/TopRanking.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Box, Flex, HStack, Image, Text, VStack } from "native-base";
+import FirstPlace from '../assets/images/firstPlace.svg';
+import SecondPlace from '../assets/images/secondPlace.svg';
+import ThirdPlace from '../assets/images/thirdPlace.svg';
+
+const TopRanking = ({ rankingList }) => {
+
+    const leafGreen = require('../assets/images/leafGreen.png');
+    
+    return (
+        <Flex direction="row" justify="center">
+          <Box paddingTop={20}>
+            <VStack alignItems="center">
+              <Text bold>#2</Text>
+              <Text>{rankingList[1].name}</Text>
+              <HStack alignItems="center">
+                <Image source={leafGreen} alt="leaf icon" size={5} resizeMode="contain" />
+                <Text bold>{rankingList[1].monthPoints? rankingList[1].monthPoints: rankingList[1].totalPoints? rankingList[1].totalPoints : 0}</Text>
+              </HStack>
+            </VStack>
+            <SecondPlace width={100} height={100} />
+          </Box>
+          <Box paddingX={3}>
+            <VStack alignItems="center">
+              <Text bold>#1</Text>
+              <Text>{rankingList[0].name}</Text>
+              <HStack alignItems="center">
+                <Image source={leafGreen} alt="leaf icon" size={5} resizeMode="contain" />
+                <Text bold>{rankingList[0].monthPoints? rankingList[0].monthPoints: rankingList[0].totalPoints? rankingList[0].totalPoints : 0}</Text>
```

Comment:
🙃

Reply:
this is because I am reusing the component for individual and departments leaderboards. One of them has monthPoints, and the other one has totalPoints. I realized that if the department exists but has no users, it doesn't contain the points attribute, in that case, its value is 0.